rhel7/templates/oval_5.11_templates/template_OVAL_service_enabled

```diff
@@ -33,12 +34,22 @@
     <linux:object object_ref="object_multi_user_target_for_%SERVICENAME%_socket_enabled" />
     <linux:state state_ref="state_systemd_%SERVICENAME%_socket_on"/>
   </linux:systemdunitdependency_test>
-
   <linux:systemdunitdependency_object id="object_multi_user_target_for_%SERVICENAME%_socket_enabled" comment="list of dependencies of multi-user.target" version="1">
     <linux:unit>multi-user.target</linux:unit>
   </linux:systemdunitdependency_object>
-
   <linux:systemdunitdependency_state id="state_systemd_%SERVICENAME%_socket_on" comment="%SERVICENAME% listed at least once in the dependencies" version="1">
     <linux:dependency entity_check="at least one">%SERVICENAME%.socket</linux:dependency>
   </linux:systemdunitdependency_state>
+
+  <linux:systemdunitproperty_test id="test_service_running_%SERVICENAME%" check="all" check_existence="at_least_one_exists" comment="Test that the %SERVICENAME% service is running" version="1">
```

Comment:
All `test_service_running_` should have `check="at least one"`. It is enough that either service or socket is in active state.